src/Xamarin.Android.Tools.AndroidSdk/Xamarin.Android.Tools.Versions.props

```diff
@@ -9,12 +9,12 @@
     If this file is changed the submodule for androidtools should be updated, 
     along with any other repo which references androidtools. 
     -->
-    <AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">29.0.2</AndroidSdkBuildToolsVersion>
+    <AndroidSdkBuildToolsVersion Condition="'$(AndroidSdkBuildToolsVersion)' == ''">28.0.3</AndroidSdkBuildToolsVersion>
     <AndroidCommandLineToolsVersion Condition=" '$(AndroidCommandLineToolsVersion)' == '' ">1.0</AndroidCommandLineToolsVersion>
-    <AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">29.0.5</AndroidSdkPlatformToolsVersion>
+    <AndroidSdkPlatformToolsVersion Condition="'$(AndroidSdkPlatformToolsVersion)' == ''">28.0.2</AndroidSdkPlatformToolsVersion>
```

Comment:
I think for the smoothest update experience, the `AndroidSdkBuildToolsVersion` and `AndroidSdkPlatformToolsVersion` can be left on the `29.0.2` and `29.0.5` versions to match d16-7:

https://github.com/xamarin/xamarin-android/blob/d16-7/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.props.in#L18-L19

Reply:
$(AndroidSdkPlatformVersion)=android-28,  is the only entry that  changed in release-8.8. We can keep other tools version as such as Brendan mentioned.

Reply:
Relevant comparison point isn't xamarin-android, but android-sdk-installer: https://github.com/xamarin/android-sdk-installer/blob/d16-7/Xamarin.Installer.AndroidSDK/Xamarin.Installer.AndroidSDK/Common/Constants.cs

…which does specify a version for `RequiredBuildToolsVersion` (29.0.2), but not for the *Platform-tools* version.

@GouriKumari? On your fresh VSMac 8.8 install, what's the output of this command?

```bash
cat ~/Library/Developer/Xamarin/android-sdk-macosx/platform-tools/source.properties
```